packages/style-engine/class-wp-style-engine.php

```diff
@@ -246,6 +252,27 @@ protected static function get_slug_from_preset_value( $style_value, $property_ke
 		return null;
 	}
 
+	/**
+	 * Generates a css var string, eg var(--wp--preset--color--background) from a preset string, eg. `var:preset|space|50`.
+	 *
+	 * @param string $style_value  A single css preset value.
+	 * @param array  $css_vars The css var patterns used to generate the var string.
+	 *
+	 * @return string|null The css var, or null if no match for slug found.
+	 */
+	protected static function get_css_var_value( $style_value, $css_vars ) {
```

Comment:
Nice. 

Could we also use this nifty function in `get_css_individual_property_rules()` below?

Reply:
Done - the unit tests tell me it is working perfectly 😄